docs/docs/community/release-process.md

```diff
@@ -0,0 +1,75 @@
+---
+id: release-process
+title: The Nebari release process
+description: The Nebari release process and workflow
+---
+
+# The Nebari release process
+
+This page describes the high-level details of cutting a new Nebari release.
+
+|                   |                                                                                                                                                                                                  |
+| :---------------: | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+|      Cadence      |                                                                                             Monthly                                                                                              |
+| Versioning System |                                                                                    [CalVer](#calver-details)                                                                                     |
```

Comment:
I know this links to the section below but I'd still spell out the CalVer flavour we use - add an example